server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/subsystem/EndpointSubsystemReader_9_0.java

```diff
@@ -597,4 +674,75 @@ private void parseSni(final XMLExtendedStreamReader reader, final ModelNode encr
       ParseUtils.requireNoContent(reader);
       operations.add(sniOp);
    }
+
+   private void parsePrefix(final XMLExtendedStreamReader reader, final ModelNode prefix, final List<ModelNode> operations) throws XMLStreamException {
```

Comment:
I'm a bit confused about what this method is really parsing... is it the prefix, the path, or both?